OpenRA.Mods.Common/Effects/AreaBeam.cs

```diff
@@ -35,6 +35,9 @@ public class AreaBeamInfo : IProjectileInfo
 		[Desc("The width of the beam.")]
 		public readonly WDist Width = new WDist(512);
 
+		[Desc("The shape of the beam.  Accepts values Cylindrical or Flat.")]
```

Comment:
It is flat on the ground, not rectangular:

![](https://cloud.githubusercontent.com/assets/167819/11460680/d0584ed6-96e8-11e5-8502-2c4bfb8afc70.png)